run.py

```diff
@@ -30,10 +28,11 @@
     )
 )
 
-df = pl.DataFrame(
-    {
-        "a": [None, None, 3, None, None, 9, 11, None],
-    }
-)
-result = df.with_columns(interpolate=mp.interpolate("a"))
-print(result)
+#df = pl.DataFrame(
+#    {
+#        "a": [None, None, 3, None, None, 9, 11, None],
+#    }
+#)
+#result = df.with_columns(interpolate=mp.interpolate("a"))
+#print(result)
```

Comment:
The PR was opened as a draft because this is now panicking, which wasn't the case before. Need to debug why this is happening now